config/source/multi/400-config-descheduler.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-descheduler
+  namespace: knative-eventing
+  labels:
+    kafka.eventing.knative.dev/release: devel
+data:
+  predicates: |+
```

Comment:
can you add an `_example` section? Here an example: https://github.com/knative/eventing/blob/main/config/core/configmaps/default-pingsource.yaml

Reply:
The `_example` checksum should also be generated. It's done in https://github.com/knative-sandbox/eventing-kafka/pull/855 which adds the necessary tooling to do so.